packages/node_modules/@webex/plugin-meetings/src/locus-Info/index.js

```diff
@@ -0,0 +1,238 @@
+        updatedParticipants = this.findChangedParticipants(participants);
+      }
+    }
+    return updatedParticipants;
+  }
+
+  // TODO: check if this causes any perfomance issue
+  findChangedParticipants(newParticipants) {
+    if (newParticipants.length < 25) {
+      const changedParticipants = [];
+      this.participants.forEach((participant) => {
+        if (!isEqual(participant, newParticipants[participant.id])) {
+          this.participants[participant.id] = newParticipants[participant.id];
+          changedParticipants[participant.id] = participant;
+        }
+      });
+
+      return changedParticipants;
+    }
+    // If more then 25 and we got fullLocus send Everything
+    return this.participants;
+    // TODO: find an alternate way or just replace the whole object
+    // Tell customers that more then 25 we will just send an update
+  }
+
+  updateControls(controls) {
+    if (controls && !isEqual(this.controls, controls)) {
+      this.controls = controls;
+      console.info('LocusInfo:controls got updated');
+    }
+  }
+
+  updateConversationUrl(conversationUrl) {
+    if (conversationUrl && !isEqual(this.conversationUrl, conversationUrl)) {
+      this.conversationUrl = conversationUrl;
+      console.info('LocusInfo:conversationUrl got updated');
+    }
+  }
+
+  updateCreated(created) {
+    if (created && !isEqual(this.created, created)) {
+      this.created = created;
+      console.info('LocusInfo:created got updated');
+    }
+  }
+
+  updateFullState(fullState) {
+    if (fullState && !isEqual(this.fullState, fullState)) {
+      this.fullState = fullState;
+      console.info('LocusInfo:fullState got updated');
+    }
+  }
+
+  updateHostInfo(host) {
+    if (host && !isEqual(this.host, host)) {
+      this.host = host;
+      console.info('LocusInfo:host got updated');
+    }
+  }
+
+  updateMeetingInfo(info) {
+    if (info && !isEqual(this.info, info)) {
+      this.info = info;
+      console.info('LocusInfo:info got updated');
+    }
+  }
+
+  updateMediaShares(mediaShares) {
+    if (mediaShares && !isEqual(this.mediaShares, mediaShares)) {
+      this.mediaShares = mediaShares;
+      console.info('LocusInfo:mediaShares got updated');
+    }
+  }
+
+  updateParticipantsUrl(participantsUrl) {
+    if (participantsUrl && !isEqual(this.participantsUrl, participantsUrl)) {
+      this.participantsUrl = participantsUrl;
+      console.info('LocusInfo:participantsUrl got updated');
+    }
+  }
+
+  updateReplace(replace) {
+    if (replace && !isEqual(this.replace, replace)) {
+      this.replace = replace;
+      console.info('LocusInfo:replace got updated');
+    }
+  }
+
+  updateSelf(self) {
+    if (self && !isEqual(this.self, self)) {
+      this.self = self;
+      console.info('LocusInfo:self got updated');
+    }
+  }
+
+  updateLocusUrl(url) {
+    if (url && this.url !== url) {
+      this.url = url;
+      console.info('LocusInfo:url got updated');
+    }
+  }
+
+  updateaclUrl(aclUrl) {
```

Comment:
typo

Reply:
there is something called acl URl i was also not sure